travis/test.sh

```diff
@@ -21,6 +21,8 @@ flutter test --flutter-repo
 
 if [ $TRAVIS_PULL_REQUEST = "false" ]; then
   if [ $TRAVIS_BRANCH = "master" ]; then
+    pub global activate dartdoc 0.8.4
+    export PATH="$PATH":"~/.pub-cache/bin"
```

Comment:
> I'm confused how we don't already have the latest version of dartdoc?

We use the version in the Dart SDK, which was frozen when the SDK was produced.  This command gives us the a newer version that was uploaded to pub.